pkg/server/devfile-handler.go

```diff
@@ -61,10 +62,15 @@ func (s *Server) devfileHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Get devfile content and parse it using a library call in the future
-	devfileContentBytes := []byte(data.Devfile.DevfileContent)
-	devfileObj, _, err = devfile.ParseDevfileAndValidate(parser.ParserArgs{Data: devfileContentBytes})
+	// devfileContentBytes := []byte(data.Devfile.DevfileContent)
+	convertUriToInlined := true
+	devfileObj, _, err = devfile.ParseDevfileAndValidate(parser.ParserArgs{
+		// Data:                          devfileContentBytes,
+		URL:                           "https://raw.githubusercontent.com/devfile-samples/devfile-sample-python-basic/main/devfile.yaml",
```

Comment:
I understand that this is WIP, but as a reminder that we don't want merge this URL unexpectedly.

Reply:
removed